source/services/notifications.md

```diff
@@ -1,18 +1,21 @@
 # Notification Service
 Allows users to send signed transaction messages between devices taking part in the signing process.
 
-[Show on GitHub](https://github.com/gnosis/safe-notification-service)
-
-[Show on Swagger](https://safe-notification.gnosis.pm/)
+[GitHub](https://github.com/gnosis/safe-notification-service)
+[Swagger](https://safe-notification.gnosis.io/)
```

Comment:
Should we add the rinkeby link here as well?

Reply:
same like for the relay below :)

Reply:
Agree :)

Reply:
I checked an relay already has the link for both and notification service is the same for rinkeby/mainnet

Reply:
I added Releases for Notification Service